new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -132,6 +135,7 @@
    */
   @BeforeAll
   public static void initAll(@Namespaces(3) List<String> namespaces) {
+    logger = ThreadSafeLogger.getLogger();
```

Comment:
you can static import ThreadSafeLogger.getLogger then just call getLogger() here like in other It* classes.